net.sourceforge.electrip.Electrip.json

```diff
@@ -22,6 +34,10 @@
                     "url": "git://git.code.sf.net/p/electrip/git",
```

Comment:
it should use https here. Which is what the linter complains about

Reply:
I can't find a https link to the code :/

Reply:
```suggestion
                    "url": "https://git.code.sf.net/p/electrip/git",
```

Reply:
Ah got it